### ISSUE TITLE
Bug: Current Podspec for Sodium/libsodium is broken

### ISSUE THREAD

**Issue by ricwein** [jedisct1/swift-sodium#75]
Since the commit 865e819616b36100318378638bfe106fbaa0e48c moves all libsodium files into a subdirectory, the Podspec should adapt the changes. Using the current Podspec will skip all files inside Sodium/libsodium directory, resulting in missing `libsodium-ios.a`/`libsodium-osx.a` and more

Instead of

```ruby
s.ios.vendored_library    = 'Sodium/libsodium-ios.a'
s.osx.vendored_library    = 'Sodium/libsodium-osx.a'

s.source_files = 'Sodium/*.{swift,h}'
```

the lines should probably be

```ruby
s.ios.vendored_library    = 'Sodium/libsodium/libsodium-ios.a'
s.osx.vendored_library    = 'Sodium/libsodium/libsodium-osx.a'

s.source_files = 'Sodium/*.{swift,h}', 'Sodium/libsodium/*.{swift,h}'
```

But even then, building with xcode for iOS fails, being unable to import libsodium.

Greetings

**Comment by blochberger** [comment]
Thanks for reporting, can you provide a pull request with the suggested changes?

What target fails to build: Your own application that links against the `Sodium` framework or the Sodium project?

**Comment by ricwein** [comment]
Added a PR #76 

Building the Sodium Target, which was added through Pods fails to build. I just created a new test-project, but still... `import libsodium` causes an "No such module 'libsodium' error.

![screen shot 2017-05-05 at 17 17 26](https://cloud.githubusercontent.com/assets/870354/25751869/c31467b8-31b6-11e7-809a-b853858016c5.png)

At least the libsodium-ios.a exists again with the changed podspec.
